app/assets/javascripts/admin/addon/services/admin-sidebar-state-manager.js

```diff
@@ -1,18 +1,28 @@
 import { tracked } from "@glimmer/tracking";
 import Service, { service } from "@ember/service";
-import { TrackedObject } from "@ember-compat/tracked-built-ins";
 import KeyValueStore from "discourse/lib/key-value-store";
 import { ADMIN_PANEL } from "discourse/lib/sidebar/panels";
 
 export default class AdminSidebarStateManager extends Service {
   @service sidebarState;
   @service currentUser;
-  @tracked keywords = new TrackedObject();
+  @tracked keywords = {};
 
   STORE_NAMESPACE = "discourse_admin_sidebar_experiment_";
 
   store = new KeyValueStore(this.STORE_NAMESPACE);
 
+  setLinkKeywords(link_name, keywords) {
```

Comment:
I just thought this was a bit more explicit than setting `adminSidebarStateManager.keywords.blah.navigation = []` directly